src/compressed_tensors/compressors/int_quantized.py

```diff
@@ -55,10 +69,7 @@ def compress(self, model_state: Dict[str, Tensor], **kwargs) -> Dict[str, Tensor
                 if scale is not None and zp is not None:
                     # weight is quantized, compress it
                     quant_args = model_quant_args[prefix]
-                    try:
-                        bit_depth = torch.finfo(value.dtype).bits
-                    except TypeError:
-                        bit_depth = torch.iinfo(value.dtype).bits
+                    bit_depth = get_torch_bit_depth(value)
                     if bit_depth > quant_args.num_bits:
```

Comment:
that example shouldn't be able to happen, since pytorch doesn't support anything below int8. I suppose though this could become an issue if num_bits was 16 and bit_depth was 8 somehow though, I can throw a warning for weird cases like that (that hopefully we should never run into)